packages/Sandblocks-Core/SBDeleteCommand.class.st

```diff
@@ -13,7 +13,7 @@ Class {
 SBDeleteCommand >> do [
 
 	| nextSelection |
-	nextSelection := target containingSandblock focusAfterDeleteIfNone: [self nextBlockFrom: target].
+	nextSelection := target containingSandblock focusAfterDeleteIfNone: [target sandblockEditor].
```

Comment:
This makes it so, that we select the editor after a top-level block deletion instead of selecting some other block (that may be far away). Is that alright?

Reply:
suggested improvement:
```
next containingFloat ~= currentFloat
		ifTrue: [target sandblockEditor]
		ifFalse: [next]
```